.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,38 @@
+## 변경 사항
+<!-- 이 PR에서 무엇을 변경했는지 간단히 설명해주세요 -->
+- 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)